src/content/whats-new/2022/01/whats-new-01-12-core-user-type.md

```diff
@@ -0,0 +1,48 @@
+---
+title: 'Introducing our new core user type
+' 
+summary: 'Unlock telemetry data access for even more developers in your org
+' 
+releaseDate: '2022-01-12' 
+learnMoreLink: 'https://example.com' 
+getStartedLink: 'https://example.com'
+---
+
+Observability is better together. Our new core user type enables more developers than ever to understand code performance better, collaborate and prioritize work directly from their IDE, and access relevant error details including stack traces, logs, alerts, and more.
+
+**New core users enable developers to:** 
+* Optimize code performance and feature planning with access to telemetry data from production and pre-production environments directly in their IDE via the [New Relic CodeStream integration.](https://newrelic.com/codestream) 
+
+* Collaborate with DevOps, site reliability engineers (SREs), and ops engineers with a direct path from [error tracking](https://newrelic.com/platform/errors-inbox) in New Relic One to the relevant code block directly in their IDE.
+* Detect patterns and outliers in [log data](https://one.newrelic.com/launcher/logger.log-launcher?platform[accountId]=3386328&platform[timeRange][duration]=1800000&platform[$isFallbackTimeRange]=true&state=18f8ab72-f351-81d7-8b9d-d0a44b5b94a3) to resolve issues and improve code performance. 
+* Build and run [custom applications](https://docs.newrelic.com/docs/new-relic-one/use-new-relic-one/build-new-relic-one/build-custom-new-relic-one-application/) in New Relic One to surface performance insights, optimize cloud costs, and more. 
+Onboard new developers with observability best practices and shared baselines for metrics.
+
+
+## Preview Period: Try it core user capabilities today
```

Comment:
@zuluecho9 I found a typo in line 22, "it" should be removed.

New copy:

## Preview period: Try core user capabilities today. `